client/admin.html

```diff
@@ -36,6 +36,7 @@ <h1 class="center">Admin Panel</h1>
 				<a href="#users" class="btn">Users</a>
 				<a href="#applicants" class="btn">Applicants</a>
 				<a href="#settings" class="btn">Settings</a>
+				<a href="#emails" class="btn">Emails</a>
```

Comment:
Please move this to after Applicants and before Settings

Reply:
Done